#control mode
uint8 control_mode
#0:rc, 1:mobile, 2:onboard
uint8 cur_ctrl_dev_in_navi_mode
#0:disabled, 1:enabled
uint8 serial_req_status
#0:disabled, 1:enabled
uint8 virtual_rc_status
